src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const table = document.body.firstElementChild;
```

Comment:
The `firstElementChild` of the body might not be the table you want. Use more specific selectors to ensure you are selecting the right table, like `getElementById` or `querySelector`.